### DISCUSSION TITLE
Script to update firefly-iii from 6.1.11 and newer

### DISCUSSION THREAD

**Discussion by pedrom34** [firefly-iii/firefly-iii#8697] [General]
Hey,

I made an update to the script I made (see #3590 )
Still, nothing fancy. Only made to make my life easier on updates.

It's available here: https://gist.github.com/pedrom34/d1b8ab84e1e9ec7e8c6cbcc3cc51d663

Hope it helps someone 👍 

Thanks James for the good work, and have a nice day 😄

**Comment by JC5** [comment]
Very cool, thank you too! 😁
